slasher/main.go

```diff
@@ -26,13 +26,25 @@ func startSlasher(ctx *cli.Context) error {
 		return err
 	}
 	logrus.SetLevel(level)
-	port := ctx.GlobalString(flags.RPCPort.Name)
+	port := ctx.GlobalInt(flags.RPCPort.Name)
 	cert := ctx.GlobalString(flags.CertFlag.Name)
 	key := ctx.GlobalString(flags.KeyFlag.Name)
+	beaconCert := ctx.GlobalString(flags.BeaconCert.Name)
```

Comment:
why do you need a beacon cert ? not sure the purpose of it

Reply:
have a way to subscribe to a certificate gated archival node